app/scripts/snaps/preinstalled-snaps.ts

```diff
@@ -6,9 +6,9 @@ import BitcoinWalletSnap from '@metamask/bitcoin-wallet-snap/dist/preinstalled-s
 ///: END:ONLY_INCLUDE_IF
 
 // The casts here are less than ideal but we expect the SnapController to validate the inputs.
-const PREINSTALLED_SNAPS: readonly PreinstalledSnap[] = Object.freeze([
-  MessageSigningSnap as unknown as PreinstalledSnap,
-  AccountWatcherSnap as unknown as PreinstalledSnap,
+const PREINSTALLED_SNAPS = Object.freeze<PreinstalledSnap[]>([
+  MessageSigningSnap as PreinstalledSnap,
+  AccountWatcherSnap as PreinstalledSnap,
```

Comment:
@k-g-j I removed an `as unknown as` cast introduced in https://github.com/MetaMask/metamask-extension/pull/26402 while merging develop into this branch.